mdm/hibernate-core/src/main/java/com/gentics/mesh/hibernate/data/dao/NodeDaoImpl.java

```diff
@@ -616,8 +616,26 @@ private Pair<Stream<NodeData>, Long> query(
 		select.setTableName(databaseConnector.maybeGetPhysicalTableName(getPersistenceClass()).get() + " " + nodeAlias);
 		List<String> columns = new ArrayList<>();
 		columns.addAll(nodeColumns);
-		maybeParents.map(parents -> makeAlias(databaseConnector.maybeGetDatabaseEntityName(HibBranchNodeParent.class).get()) + ".*").ifPresent(columns::add);
-		Optional.of(noContainerEdgeFetch).filter(noFetch -> !noFetch).flatMap(fetch -> maybeContainerLanguages).or(() -> maybeContentColumns.map(contentColumns -> Collections.emptyList())).map(yes -> makeAlias("CONTAINER") + ".*").ifPresent(columns::add);
+
+		// Ask for the parent edge columns, if parent/chilren requested
+		maybeParents.ifPresent(parents -> {
```

Comment:
Star column fetches now behave awkward, if more than one entity type is requested.